package.sh

```diff
@@ -157,16 +163,25 @@ else
 	##    - building binaries from sources
 	##    - copy artifacts to the build_dir
 	echo "Build the module..."
-	if [ -e ./packaging/build.sh ]; then
-		./packaging/build.sh $PWD ${build_dir} || exit 1
-	else
+	if [ ! -e ./packaging/build.sh ]; then
 		echo "ERROR: No build script available"
 		exit 1
 	fi
+	if [ "${kernel_config}" == "" ]; then
+		./packaging/build.sh $PWD ${build_dir} || exit 1
+	else
+		### Kernel build needs to know which configuration to be built.
+		./packaging/build.sh $PWD ${kernel_config} || exit 1
+	fi
 
 	if [ -e ./packaging/package.sh ]; then
 		echo "INFO: Use package script provided by module."
-		./packaging/package.sh $PWD || exit 1
+		if [ "${kernel_config}" == "" ]; then
+			./packaging/package.sh $PWD || exit 1
```

Comment:
Same here, kernel_config could be just second param. package.sh can either use it or not.

Reply:
ok. It will be changed.